test/tests.jl

```diff
@@ -42,7 +42,9 @@ end
         @test isdefined(methods(M.j_1), :mt)
         local mt = methods(M.j_1).mt
         @test isa(mt, Core.MethodTable)
-        @test isdefined(mt, :kwsorter)
+        if hasfield(Core.MethodTable, :kwsorter)
```

Comment:
Julia 1.0 support...

```suggestion
        if fieldindex(Core.MethodTable, :kwsorter, false) > 0
```